build/gtk-bundle-from-msys2.sh

```diff
@@ -0,0 +1,259 @@
+#!/bin/sh
+#
+# Fetch and extract Geany-Plugins dependencies for Windows/MSYS2
+# This script will download (or use Pacman's cache) to extract
+# plugin dependencies as defined below. To be run within a MSYS2
+# shell. The extracted files will be placed into the current
+# directory.
+
+ABI=i686
+use_cache="no"
+make_zip="no"
+gtkv="3"
+
+# ctags - binary for GeanyCTags plugin
+# ctpl-git - for GeanyGenDoc plugin
+# enchant, hunspell - for SpellCheck plugin
+# curl, glib-networking, gnutls, icu, sqlite3, webkitgtk2/3 for WebHelper and Markdown plugins
+# lua51 - for GeanyLua plugin
+# gnupg, gpgme - for GeanyPG plugin
+# libsoup - for UpdateChecker plugin
+# libgit2 - for GitChangeBar plugin
+# libxml2 - for PrettyPrinter plugin
+# gtkspell - for GeanyVC plugin
+# the rest is dependency-dependency
```

Comment:
can't pacman be used to resolve dependencies of deps?  would look more robust if it could